### ISSUE TITLE
netstat command not found, on CentOS / RHEL7

### ISSUE THREAD

**Issue by richardforth** [richardforth/apache2buddy#17]
[ INFO     ] We are checking the service running on port 80...
sh: netstat: command not found
[ WARNING  ] Nothing seems to be listening on port 80. Falling back to process list...
sh: netstat: command not found
Use of uninitialized value $real_port in concatenation (.) or string at - line
	1429 (#1)
    (W uninitialized) An undefined value was used as if it were already
    defined.  It was interpreted as a "" or a 0, but maybe it was a mistake.
    To suppress this warning assign a defined value to your variables.
    
    To help you figure out what was undefined, perl will try to tell you
    the name of the variable (if any) that was undefined.  In some cases
    it cannot do this, so it also tells you what operation you used the
    undefined value in.  Note, however, that perl optimizes your program
    anid the operation displayed in the warning may not necessarily appear
    literally in your program.  For example, "that $foo" is usually
    optimized into "that " . $foo, and the warning will refer to the
    concatenation (.) operator, even though there is no . in
    your program.

**Comment by richardforth** [comment]
The netstat command was deprecated in RHEL 7 and its variants. Looking to see if I can establish an easy fix...eg missing package that might provide this.

I know we have the 'ss' command now but the output is vastly different.

**Comment by richardforth** [comment]
apparently the net-tools package provides this, will test.

**Comment by richardforth** [comment]
The 'net-tools' package does indeed provide this tool.

I will add a pre-flight check to check for the existence of this package and an appropriate warning and helpful hint.

**Comment by richardforth** [comment]
Fix applied to staging:

when working OK:

```
Performing Auto-Discovery, and Pre-Flight Checks...
[ OK       ] This script is being run as root.
[ OK       ] The utility 'pmap' exists and is available for use: /bin/pmap
[ OK       ] The utility 'netstat' exists and is available for use: /bin/netstat
[ OK       ] 'php' exists and is available for use: /bin/php
```

When netstat is not found:

```
Performing Auto-Discovery, and Pre-Flight Checks...
[ OK       ] This script is being run as root.
[ OK       ] The utility 'pmap' exists and is available for use: /bin/pmap
which: no netstat in (/usr/local/sbin:/sbin:/bin:/usr/sbin:/usr/bin:/root/bin)
[ CRITICAL ] Unable to locate the netstat utility. This script requires netstat to determine the port that apache is listening on.
[ INFO     ] To fix this make sure the net-tools package is installed.
```

**Comment by richardforth** [comment]
I know that netstat is decprecated and will probably be removed in RHEL 8, in favour of the ss command, but right now this is too much work to re-code and cater for.

**Comment by richardforth** [comment]
merged #17
